docs/references/experimental/monorepo/README.md

```diff
@@ -0,0 +1,42 @@
+## Monorepo Projects
+
+### What is it?
+
+FOSSA experimentally supports scanning large monorepo projects which have potentially many licenses spread out across its files. The monorepo feature also supports inferring build graph information depending on the type of monorepo project.
+
+Currently, the only supported monorepo type is the Android Open Source Project ([AOSP](#AOSP)).
+
+Currently, this also disables all other FOSSA strategies, and therefore does not do any dependency analysis.
+
+### Prerequisites
+
+Monorepo support requires a feature flag enabled in your FOSSA organization. If you're interested in using this feature please contact us!
+
+### Interacting with Monorepo Scans
+
+When viewing a Monorepo project in the FOSSA UI, users will a list of files and directories with metadata that was collected during the scan. In this UI, files can be filtered by metadata such as (but not limited to)
```

Comment:
Bit confusing, active voice might be better for this sentence.

Maybe something along the line of: FOSSA UI will list scanned files and directories. Each file will be associated with the metadata collected during the scan. In the UI, files can be ....